mantidproject/mantid
# fix I(Q,t) over IN16B data

**Description of work.**

Fixes the backend for I(Q,t) GUI to properly identify the run numbers when using IN16B data.

**Report to:** Filippa Lundin/ESS. 

**To test:**

1. Go to Interfaces->Indirect->Data Analysis->I(Q,t)
2. Use the attached files 235K for sample, 2K for resolution.
3. Click Run, this should produce an _iqt workspace, and no errors. (there will be a warning though, which is a separate issue).

Try once with the files, and once with the workspaces, i.e. load the files first, and then choose the workspaces in the interface.



No issue number.


Chop off the `.zip`, those are nexus files!

[239349_235K_red.nxs.zip](https://github.com/mantidproject/mantid/files/2643304/239349_235K_red.nxs.zip)
[239281_2K_red.nxs.zip](https://github.com/mantidproject/mantid/files/2643305/239281_2K_red.nxs.zip)



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
